### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by ShahaamMarzook**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Fix Travis errors

**Comment by ShahaamMarzook** [comment]
Direct me how to update blog in this so that I could finish my task GCI17

**Review by ShahaamMarzook — COMMENTED** [review]
Is this correct 
Or what should I do to update the blog

File: _data/blogs.yml [hunk 1]
```diff
@@ -272,3 +272,8 @@
   title: GCI 2017 with FOSSASIA
   img: axirygit.jpg
   author_name: Eiffel Chong
+  
+  - url: https://workingwithfossasiawordpress.wordpress.com/2018/01/06/participation-in-open-source-community/
```
Comment (abishekvashok) [inline _data/blogs.yml:276]: No spaces before -

**Review by abishekvashok — CHANGES_REQUESTED** [review]
The image should be 240x240px

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please make the above changes and upload the image in the img/blogs folder

**Review by isuruAb — COMMENTED** [review]
please resolve Travis failure

**Review by abishekvashok — CHANGES_REQUESTED** [review]
You need to upload the image as well under the name `Blog pic.jpeg` to img/blogs/

**Review by ParthS007 — COMMENTED** [review]
@ShahaamMarzook Closing this PR due to many conflicts and inactivity.Please read `README.MD` on how to add a blog and make a PR.Please ask on FOSSASIA Gitter channel if you face any difficulty 👍

**Pull request closed without merging**